tests/test_app.py

```diff
@@ -124,6 +124,15 @@ def _log_in_user(self):
             follow_redirects=False
         )
 
+    @mock.patch('requests.get', return_value=fake_title)
+    def test_postcode_search_success(self, mock_get):
+        response = self.app.post('/title-search/', data=dict(search_term='LU1 1DZ'), follow_redirects=True)
+        assert response.status_code == 200
+        assert 'DN1000' in str(response.data)
```

Comment:
Best to use `response.data.decode()`. `str(response.data)`, while very similar, is not the HTML we expect it to be. It may contain escaped characters and is wrapped in `b'...'`